Sources/Starknet/Data/TypedData/TypeDeclaration.swift

```diff
@@ -98,5 +98,17 @@ public extension StarknetTypedData {
                 self = try .standard(StandardType(from: decoder))
             }
         }
+
+        public func encode(to encoder: Encoder) throws {
```

Comment:
Does this have to be public?

Reply:
Yes because it matches a requirement in public protocol `Encodable`.